playbooks/robusta_playbooks/k8s_resource_enrichments.py

```diff
@@ -188,12 +247,20 @@ def related_pods(event: KubernetesResourceEvent, params: RelatedPodParams):
 
     Supports Deployments, ReplicaSets, DaemonSets, StatefulSets and Pods
     """
-    pods = get_related_pods(event.get_resource())
     cluster = event.get_context().cluster_name
 
     if params.output_format == "json":
-        event.add_enrichment([JsonBlock(json.dumps([to_pod_obj(pod, cluster).dict() for pod in pods]))])
+        pods = get_related_pods(event.get_resource(), limit=params.limit)
+        event.add_enrichment([JsonBlock(json.dumps([to_pod_obj(pod, cluster, include_raw_data=params.include_raw_data).dict() for pod in pods], default=str))])
+    elif params.output_format == "json2":
```

Comment:
I know I added it, but you can remove this json2 code! It wasn't used in the end.